### PR TITLE
LPS-149120

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related Issue: https://issues.liferay.com/browse/LPS-149120

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:stop

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @p-albuquerque, can you please check if these failures are related to your changes? Thanks

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @p-albuquerque, can you please check if these failures are related to your changes? Thanks

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @p-albuquerque, can you please check if these failures are related to your changes? Thanks

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Review by mateussgn — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/view/table/ObjectEntriesTableFDSView.java [hunk 1]
```diff
@@ -101,7 +101,17 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
...
+					String label = objectViewColumn.getLabel(locale);
+
+					if (label.isEmpty()) {
+						_addObjectField(
+							fdsTableSchemaBuilder, objectField,
+							objectField.getLabel(locale));
+					}
+					else {
+						_addObjectField(
+							fdsTableSchemaBuilder, objectField, label);
```
[the remark this answers is not in the document]
Comment (p-albuquerque) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/view/table/ObjectEntriesTableFDSView.java]: Ok. Done

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Review by mateussgn — APPROVED** [review]
LGTM! 🎉

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**